src/lib/holocene/select/option-group.svelte

```diff
@@ -1,5 +1,5 @@
 <script lang="ts">
-  import MenuItem from '../primitives/menu/menu-item.svelte';
+  import MenuItem from '$lib/holocene/menu/menu-item.svelte';
 
   export let label: string;
```

Comment:
Wondering if we want a little bit of an indent between the label and the options?

![Screenshot 2023-08-18 at 11 01 25 AM](https://github.com/temporalio/ui/assets/15069288/74b76515-64b3-4898-bb6f-eccaaeb7bfbf)

Reply:
We currently have any Selects with grouped options, and Select will be deprecated soon in favor of Combobox, so I'm not too worried about this.